docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/install.md

```diff
@@ -705,51 +729,41 @@ helm upgrade --values spectro-mgmt-plane/values.yaml image-swap image-swap-0.0.0
 
    :::
 
-9. Use the custom domain name or the IP address of the load balancer to visit the Palette system console. To access the
-   system console, open a web browser and paste the custom domain URL in the address bar and append the value `/system`.
-   Replace the domain name in the URL with your custom domain name or the IP address of the load balancer.
-   Alternatively, you can use the load balancer IP address with the appended value `/system` to access the system
-   console.
+10. Use the custom domain name or the IP address of the load balancer to visit the Palette system console. To access the
+    system console, open a web browser and paste the custom domain URL in the address bar and append the value
+    `/system`. Replace the domain name in the URL with your custom domain name or the IP address of the load balancer.
+    Alternatively, you can use the load balancer IP address with the appended value `/system` to access the system
+    console.
 
-The first time you visit the Palette system console, a warning message about a not trusted SSL certificate may appear.
-This is expected, as you have not yet uploaded your SSL certificate to Palette. You can ignore this warning message and
-proceed.
+    The first time you visit the Palette system console, a warning message about a not trusted SSL certificate may
+    appear. This is expected, as you have not yet uploaded your SSL certificate to Palette. You can ignore this warning
+    message and proceed.
```

Comment:
Do you think it would make sense to link to vertex/system-management/ssl-certificate-management.md here?